source/guides/maintenance/custom-app-container.rst

```diff
@@ -0,0 +1,402 @@
+.. _custom-app-container:
+
+Create customized application container images
+##############################################
+
+This guide describes how to edit |CL-ATTR| container base images to customize
+an application container image.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Overview
+********
+
+The official |CL-ATTR| container base images are published on Docker\* Hub and
+are updated on a regular basis. Popular applications that use |CL| are also
+published as `Docker images on Docker Hub`_. The source for these Docker images
+is available on `GitHub`_\*.
+
+In this guide, the phrase *Clear Linux\* OS container base images* is used to
+refer to the official `clearlinux image on Docker Hub`_. This guide describes how
+to edit |CL| container base images for the following situations:
+
+
+*	Customize an application image Dockerfile by adding a bundle
+*	Rebuild a published Clear Linux OS Dockerfile with desired version
+
+ 	*	Single-stage Clear Linux OS Dockerfiles
+	*	Multi-stage Clear Linux OS Dockerfiles
+
+*	Customize an application image at runtime
+
+
+Prerequisites
+*************
+
+*	Set up a functional Docker environment as described in :ref:`docker`.
+*	Download the |CL| microservice Dockerfile repo with the following command:
+
+ 	.. code-block:: bash
+
+ 		git clone https://github.com/clearlinux/dockerfiles.git
+
+Background
+**********
+

```

Comment:
Take the first sentence of this para ("Most of the images...) and include it in the previous section. 
Then move "Background" section to appear just before `Related topics` toward bottom.  As a guide, it should be easy for users to try an example quickly without having to read the entire background first.